test/src/main/configurations/HTTP/ConfigurationApiListener.xml

```diff
@@ -9,7 +9,7 @@
 				name="SimpleInsertListener"
 				className="org.frankframework.http.rest.ApiListener"
 				uriPattern="insert"
-				method="post"
+				methods="post,get"
```

Comment:
Just to validate that setting multiple values does work in real life.

Reply:
Misschien is het handiger om PUT te gebruiken 😄

Reply:
Changed it.